src/styles/custom.css

```diff
@@ -1826,12 +1826,12 @@ to be custom HTML.
 }
 
 .truncate-3 {
-    line-height: 1.4;
-    display: -webkit-box;
-    -webkit-line-clamp: 3;
-    -webkit-box-orient: vertical;
-    overflow: hidden;
-    text-overflow: ellipsis;
+	line-height: 1.4;
```

Comment:
@ShahAnuj2610 Why are mixing spaces and tabs indent?

Reply:
it's due to prettier